examples/dax/cluster.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: dax.aws.crossplane.io/v1alpha1
+kind: Cluster
+metadata:
+  name: example-dax-cluster
+spec:
+  forProvider:
+    description: "This is an example dax cluster."
+    tags:
+      - key: "testKey"
+        value: "testValue"
+    sseSpecification:
+      enabled: true
+    securityGroupIDs:
```

Comment:
can we use here `securityGroupIdRef` ?

Reply:
That sounds good, I've changed this to `securityGroupIdRef` in [`0aac4`](https://github.com/crossplane/provider-aws/pull/1256/commits/0aac42d58a30c4e20b55d6810dc00363ba1ab2c2).